src/main/telemetry/jetiexbus.c

```diff
@@ -299,32 +387,37 @@ void handleJetiExBusTelemetry(void)
     }
 }
 
-void sendJetiExBusTelemetry(uint8_t packetID)
+
+
+uint8_t sendJetiExBusTelemetry(uint8_t packetID, uint8_t item)
 {
-    static uint8_t sensorDescriptionCounter = 0;
-    static uint8_t sensorValueCounter = 1;
+    static uint8_t sensorDescriptionCounter = 0xFF;
     static uint8_t requestLoop = 0;
     uint8_t *jetiExTelemetryFrame = &jetiExBusTelemetryFrame[EXBUS_HEADER_DATA];
-
-    if (requestLoop == 100) {              //every nth request send the name of a value
-        if (sensorDescriptionCounter == JETI_EX_SENSOR_COUNT )
-            sensorDescriptionCounter = 0;
-
+    
+    if (requestLoop < 0xFF) {
+        while(1) {
+            sensorDescriptionCounter++;
+            if(jetiExSensors[sensorDescriptionCounter].parameter >= EX_TYPE_SENSOR_ENABLED) break;
+            if (sensorDescriptionCounter == JETI_EX_SENSOR_COUNT ) {
+                sensorDescriptionCounter = 0;
+                break;
+            }
+        }
         createExTelemetrieTextMessage(jetiExTelemetryFrame, sensorDescriptionCounter, &jetiExSensors[sensorDescriptionCounter]);
         createExBusMessage(jetiExBusTelemetryFrame, jetiExTelemetryFrame, packetID);
-
-        requestLoop = 0;
-        sensorDescriptionCounter++;
+        requestLoop++;
     } else {
-        sensorValueCounter = createExTelemetrieValueMessage(jetiExTelemetryFrame, sensorValueCounter);
+        if(item == activeSensors) item = 0;
+        item = createExTelemetrieValueMessage(jetiExTelemetryFrame, item);
```

Comment:
Can we please rename this to `createExTelemetryValueMessage` for consistency?

Reply:
Is renamed.